.github/workflows/build.yml

```diff
@@ -221,15 +223,27 @@ jobs:
           - os: ubuntu-latest
             architecture: linux-aarch64
             plat-name: manylinux2014_aarch64
+            environment:
+              name: pypi
+              url: https://pypi.org/p/aries-askar
```

Comment:
This should not be aries-askar.  Ideally it is just “askar” — but lets find out if that is taken.  If it is, perhaps “owf-askar”?

Reply:
Oh. Changing the project and package name is a lot more work. aries-askar is in a tons of places. This was just trying to get a release published with the existing project.

Reply:
Adding the comment again here so it is in the right place.  Sounds good — let’s leave the name as is for now.  We can talk about a change later.